sw/otbn/crypto/p256_base.s

```diff
@@ -1580,7 +1580,7 @@ p256_random_scalar:
      w16 <= w16[63:0] */
   bn.rshi   w18, w31, w16 >> 192
   bn.rshi   w20, w16, w31 >> 64
-  bn.rshi   w16, w20, w31 >> 192
+  bn.rshi   w16, w31, w20 >> 192
```

Comment:
LGTM because `w31` (which is all zero) forms the upper part of the concatenation before the right-shift, `w20` contains `w16[63:0]` in the upper-most 64 bit, and per the comment above the assignment should be `w16 <= w16[63:0]`.